apis/python/test/test_ingestion.py

```diff
@@ -19,6 +20,271 @@ def query_and_check_equals(index, queries, expected_result_d, expected_result_i)
     check_equals(result_d=result_d, result_i=result_i, expected_result_d=expected_result_d, expected_result_i=expected_result_i)
 
 
+
+def test_ivf_flat_ingestion_with_updates_and_timetravel(tmp_path):
```

Comment:
Please move this back to its previous place in the file so that the diff can be smaller and ensure that there is no change in the test code